index.go

```diff
@@ -94,6 +94,7 @@ const indexTmpl string = `
       {{end}}
 
       window.ui = ui
+      document.getElementsByTagName("body")[0].style="margin:0;padding:0" 
```

Comment:
@projectxiel Why not with native style tag?

Reply:
I tried that when forking, and couldn't quit get it to work, maybe someone else can